show/plugins/sonic-system-ldap_yang.py

```diff
@@ -0,0 +1,153 @@
+"""
+Auto-generated show CLI plugin.
+
+
+"""
+
+import click
+import tabulate
+import natsort
+import utilities_common.cli as clicommon
+
+
+
+
+
+def format_attr_value(entry, attr):
+    """ Helper that formats attribute to be presented in the table output.
+
+    Args:
+        entry (Dict[str, str]): CONFIG DB entry configuration.
+        attr (Dict): Attribute metadata.
+
+    Returns:
+        str: fomatted attribute value.
+    """
+
+    if attr["is-leaf-list"]:
+        return "\n".join(entry.get(attr["name"], []))
+    return entry.get(attr["name"], "N/A")
+
+
+@click.group(name="ldap-server",
+             cls=clicommon.AliasedGroup,
+             invoke_without_command=True)
+@clicommon.pass_db
+def LDAP_SERVER(db):
+    """  [Callable command group] """
+
+    header = [
+"IPADDRESS",
+
+"PRIORITY",
+
+]
+
+    body = []
+
+    table = db.cfgdb.get_table("LDAP_SERVER")
+    for key in natsort.natsorted(table):
+        entry = table[key]
+        if not isinstance(key, tuple):
+            key = (key,)
+
+        row = [*key] + [
+    format_attr_value(
+        entry,
+        {'name': 'priority', 'description': 'Server priority', 'is-leaf-list': False, 'is-mandatory': False, 'group': ''}
+    ),
+]
+
+        body.append(row)
+
+    click.echo(tabulate.tabulate(body, header))
+
```

Comment:
Auto generated code, refactor and code clean up must be done

Reply:
The code was autogenerated by using `sonic-cli-gen` tool
fixed spaces as you mentioned,
pls comment if there is some other issue.
thanks